luarules/gadgets/game_team_power_watcher.lua

```diff
@@ -62,233 +29,462 @@ if gadgetHandler:IsSyncedCode() then
         {threshold = 475000, techLevel = 3.5},
         {threshold = 600000, techLevel = 4},
         {threshold = 725000, techLevel = 4.5}
-    }
+    },
+
+
+    --stores the values of powers for teams
```

Comment:
Delete comment, code is self-explanatory

Reply:
Yep